### PR TITLE
update personas

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@KaitCrawford @TrevorOctober review please.
This adds new personaes for the UG instance, overriding the default personaes in IoGT

File: springboard_iogt_ug/views.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+from springboard_iogt.utils import (
+    get_redirect_url)
+
+PERSONAE = {'YOUTH_BOY', 'YOUTH_GIRL', 'FATHER', 'MOTHER'}
+
+
+class UGIogtViews(IoGTViews):
+    @ga_context(lambda context: {'dt': 'Selected Persona', })
+    @view_config(route_name='select_persona')
+    def select_persona(self):
```
Comment (KaitCrawford) [inline springboard_iogt_ug/views.py:15]: Do method decorators work on class methods? I read something last night that said they don't. How are these decorators?
https://docs.djangoproject.com/en/1.8/topics/class-based-views/intro/#id2

**Comment by KaitCrawford** [comment]
:+1: Just left a question for my own learning

File: springboard_iogt_ug/views.py (hunk 1 above)
Comment (miltontony) [inline springboard_iogt_ug/views.py:15]: Yeah, you can, with a little bit more effort. The issue is often that a class method's first argument is `self` so for non-class-method decorators, you can't just use them as is. If you're writing your own, you can easily cater for that.

**Pull request merged**